src/LoginHandler.php

```diff
@@ -35,10 +36,22 @@ public function __construct()
          */
         add_action('login_footer', function (): void {
             $html = ob_get_clean();
+
+            if (EnforceSingleSignOn::is_active()) {
+                // Also clean up the HTML by removing the login form itself
+                $html = preg_replace('/<form[^>]*id=["\']loginform["\'][^>]*>.*?<\/form>/is', '', $html);
```

Comment:
Hiding the form completely is a nice touch, but not sure if it's an absolute safeguard for not authenticating with username/password. What do you think @sagarsdeshmukh?

There must be a a way to tap into WP's `authenticate` and block normal authentication completely.